cmd/sync.js

```diff
@@ -136,9 +140,19 @@ const handler = function(argv) {
         process.exit(1)
       }
 
+      // converting comma separated list of labels to array
+      const labels = argv.labels ? argv.labels.split(',') : ['trira']
```

Comment:
This is wrong - what if I don't want to push any labels?

Reply:
I left default value because there were default value in previous implementation (I just changed it from 'test-case' to 'trira'. You are right that there is no way to push no label. I will remove the default value altogether.